source/android/Renaissance/lib/home_page.dart

```diff
@@ -9,14 +13,238 @@ class HomePage extends StatefulWidget {
 }
 
 class _HomePageState extends State<HomePage> {
+  var _pageIndex = 0;
+
+  void changePage(int num) {
+    setState(() {
+      _pageIndex = num;
+      print("Deneme");
```

Comment:
These print lines should be removed

Reply:
Okay, the print statements were for debugging.